### PR TITLE
adding non-root user to dockerfile

### PR DESCRIPTION
**Pull request by vsamidurai**
adding non-root user to kubefed controller dockerfile #929

### CODE REVIEW

File: images/kubefed/Dockerfile [hunk 1]
```diff
@@ -15,11 +15,13 @@
 
 FROM alpine:latest
 RUN apk --no-cache add ca-certificates
+RUN adduser -D hyperfed
 
-WORKDIR /root/
+WORKDIR /hyperfed/
```
Comment (marun) [inline images/kubefed/Dockerfile]: Should this be /home/hyperfed ?  Or should this directory be created and owned by the hyperfood user?

Comment (vsamidurai) [reply images/kubefed/Dockerfile]: directory should be owned by hyperfed user. updated docker file with `chown`

**Comment by marun** [comment]
@vsamidurai You'll also need to update the deployment to account for the new location for the `controller-manager` command:

https://github.com/kubernetes-sigs/kubefed/blob/master/charts/kubefed/charts/controllermanager/templates/deployments.yaml#L27

**Comment by vsamidurai** [comment]
Thank you @marun. I updated the helm chart template `controller-manager` and `webhook` command.

**Comment by marun** [comment]
Thank you!

/approve
/lgtm

**Pull request merged**